explorer/interpreter/type_checker.cpp

```diff
@@ -2095,15 +2095,15 @@ class TypeChecker::SubstitutedGenericBindings {
 };
 
 auto TypeChecker::Substitute(const Bindings& bindings,
-                             Nonnull<const Value*> type) const
```

Comment:
In Carbon, types are values, so I'm not sure this is wrong.

Reply:
It's just that the non-defining declaration in the header calls this parameter `value` (which clang-tidy points out). We could change the header instead, but the name is already incorporated in the documentation there. Thoughts?

Reply:
OK, fair enough.